### PR TITLE
Change Getting started Use case images

### PR DESCRIPTION
**Pull request by javimed**
## Description

Replace and add Wazuh's dashboard images in Getting started > Use cases.

## Checks
- [ ] It compiles without warnings.
- [X] Spelling and grammar. 
- [X] Used impersonal speech. 
- [X] Used uppercase only on nouns. 
- [X] Updated the `redirect.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).

### CODE REVIEW

File: source/getting-started/use_cases/cloud_security.rst [hunk 1]
```diff
@@ -99,8 +105,10 @@ Expand the output to see an example of an alert generated when an AWS security g
 
 Example of AWS dashboard:
 
-.. thumbnail:: ../../images/getting_started/use_case_cloud.png
+.. thumbnail:: /images/getting_started/use-cases/wazuh-use-cases-cloud-geo.png
```
Comment (MarVergara) [inline source/getting-started/use_cases/cloud_security.rst:108]: This image needs to be edited. I can see the black line from the header you removed.

image

File: source/getting-started/use_cases/configuration_assessment.rst [hunk 2]
```diff
@@ -71,8 +71,25 @@ macOS SCA rule example:
 
 Below is an example of the results of a configuration assessment evaluation. These can be obtained through the web user interface or directly through the Wazuh RESTful API.
 
-.. thumbnail:: ../../images/getting_started/use_case_sca.png
+.. thumbnail:: /images/getting_started/use-cases/wazuh-use-cases-sca1.png
```
Comment (MarVergara) [inline source/getting-started/use_cases/configuration_assessment.rst:74]: These images need to be edited. I can see the black line from the header you removed. 

image

File: source/getting-started/use_cases/file_integrity.rst [hunk 3]
```diff
@@ -98,9 +98,29 @@ Expand the output to see an example response of this use case.
 
 In the *Integrity monitoring* module of the Wazuh dashboard, users can view all of the details of the alerts triggered and find a comprehensive summary of changes in files and registry keys. 
 
+.. raw:: html
```
Comment (MarVergara) [inline source/getting-started/use_cases/file_integrity.rst:101]: These images need to be edited. I can see the black line from the header you removed.

Also, I can tell images in this class have different sizes. Please, ask the design team to adjust them properly. If this is too complicated for them to change, we can discuss it we should leave them as it is. 

image

File: source/getting-started/use_cases/file_integrity.rst [hunk 4]
```diff
@@ -98,9 +98,29 @@ Expand the output to see an example response of this use case.
...
+.. thumbnail:: /images/getting_started/use-cases/wazuh-use-cases-fim1.png
+      :title: Integrity monitoring dashboard
+      :align: center
+
+.. thumbnail:: /images/getting_started/use-cases/wazuh-use-cases-fim2.png
+      :title: Integrity monitoring
+      :align: center
+
+..
+   .. thumbnail:: /images/getting_started/use-cases/wazuh-use-cases-fim3.png
```
Comment (MarVergara) [inline source/getting-started/use_cases/file_integrity.rst:115]:
```suggestion

```

File: source/getting-started/use_cases/file_integrity.rst [hunk 5]
```diff
@@ -98,9 +98,29 @@ Expand the output to see an example response of this use case.
...
+          :title: Integrity monitoring inventory
```
Comment (MarVergara) [inline source/getting-started/use_cases/file_integrity.rst:116]:
```suggestion

```

File: source/getting-started/use_cases/log_analysis.rst [hunk 6]
```diff
@@ -79,11 +79,24 @@ Expand the output to see an example found in ``/var/ossec/logs/alerts/alerts.jso
...
 
-Once generated by the :ref:`Wazuh server <wazuh_server>`, the alerts are sent to the :ref:`Wazuh indexer <wazuh_indexer>` component where they are enriched with geolocation information, stored, and indexed. Kibana can then be used to search, analyze, and visualize the data. Below is an alert as displayed in the interface:
+Once generated by the :ref:`Wazuh server <wazuh_server>`, the alerts are sent to the :ref:`Wazuh indexer <wazuh_indexer>` component where they are enriched with geolocation information, stored, and indexed. Kibana can then be used to search, analyze, and visualize the data.
 
-.. thumbnail:: ../../images/getting_started/use_cases_log_analysis.png
-   :title: Security events
-   :align: center
-   :wrap_image: No
+
+.. raw:: html
```
Comment (MarVergara) [inline source/getting-started/use_cases/log_analysis.rst:85]: This case is very subtle so I understand if you consider there's no need to modify these images, but I do see a very small black line from the header you removed.


image

File: source/getting-started/use_cases/regulatory_compliance.rst [hunk 7]
```diff
@@ -97,11 +97,28 @@ Expand the output to see an example alert for rule ``Apache: Attempt to access f
     "timestamp": "2020-09-04T06:08:53.878+0000"
   }
 
-Example of regulatory compliance dashboard for PCI DSS:
-
-.. thumbnail:: ../../images/getting_started/use_case_regulatory_compliance.png
-   :align: center
-   :wrap_image: No
+Example of regulatory compliance dashboards:
```
Comment (MarVergara) [inline source/getting-started/use_cases/regulatory_compliance.rst:100]: These images need to be edited. I can see the black line from the header you removed.

Also, I can tell images in this class have different sizes. Please, ask the design team to adjust them properly. If this is too complicated for them to change, we can discuss it we should leave them as it is. 


image

File: source/getting-started/use_cases/security_analytics.rst [hunk 8]
```diff
@@ -22,21 +22,12 @@ Below you can see a list of these components:
...
-In addition, these tools assist users in detecting the possible presence of rootkits and collecting inventory data from different sources such as operating systems, processes, applications, ports, and network interfaces. FIM results can be observed in the **Integrity monitoring** module dashboard of the :ref:`Wazuh Kibana plugin<kibana_app>`.
-
-.. thumbnail:: ../../images/getting_started/integrity_monitoring_1.png 
-  :title: FIM overview
-  :align: center
-  :width: 100%
-
-.. thumbnail:: ../../images/getting_started/integrity_monitoring_2.png 
-  :title: FIM dashboard
+.. thumbnail:: /images/getting_started/use-cases/wazuh-use-cases-security-analytics.png
```
Comment (MarVergara) [inline source/getting-started/use_cases/security_analytics.rst:25]: Why does the screenshot of this dashboard look cut at the end?
Was this a mistake? 

image

File: source/getting-started/use_cases/vulnerability_detection.rst [hunk 9]
```diff
@@ -88,8 +88,12 @@ Expand the output to see an example of a vulnerability detection alert.
 
 Vulnerability detection dashboard:
 
-.. thumbnail:: ../../images/getting_started/use_case_vulnerability.png
-   :align: center
-   :wrap_image: No
+.. thumbnail:: /images/getting_started/use-cases/wazuh-use-cases-vuln1.png
```
Comment (MarVergara) [inline source/getting-started/use_cases/vulnerability_detection.rst:91]: This structure (one image above the other) doesn't look good. 
I prefer adding a `class ` here even though they have different sizes if there's no other way.
Also, these images need to be edited. I can see the black line from the header you removed. 
image

**Review by MarVergara — CHANGES_REQUESTED** [review]
I requested some changes for you to review or consider.

File: source/getting-started/use_cases/file_integrity.rst (hunk 4 above)
Comment (javimed) [reply source/getting-started/use_cases/file_integrity.rst:115]: Lines 115 and 116 were not removed but commented to let wazuh-use-cases-fim3.png ready to be included in a future revision.

File: source/getting-started/use_cases/file_integrity.rst (hunk 5 above)
Comment (javimed) [reply source/getting-started/use_cases/file_integrity.rst:116]: Lines 115 and 116 were not removed but commented to let wazuh-use-cases-fim3.png ready to be included in a future revision.

**Comment by FelipeKnudsen** [comment]
Team, I attach the resized images that you need, let me know if those are ok, or you need something else. Thanks!

**Review by MarVergara — APPROVED** [review]

**Review by MarVergara — APPROVED** [review]
LGTM

**Pull request merged**